Modules/EMCAL/src/RawCheck.cxx

```diff
@@ -80,6 +150,54 @@ void RawCheck::beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult)
     }
     h->SetLineColor(kBlack);
   }
+  if (mo->getName() == "BunchMinRawAmplutudeFull_PHYS") {
+    auto* h = dynamic_cast<TH1*>(mo->getObject());
+    TPaveText* msg = new TPaveText(0.5, 0.5, 0.9, 0.75, "NDC");
+    h->GetListOfFunctions()->Add(msg);
+    msg->SetName(Form("%s_msg", mo->GetName()));
+
+    if (checkResult == Quality::Good) {
+      msg->Clear();
+      msg->AddText("No Error: OK!!!");
+      msg->SetFillColor(kGreen);
+      h->SetFillColor(kGreen);
+    } else if (checkResult == Quality::Bad) {
+      LOG(info) << "Quality::Bad, setting to red";
+      msg->Clear();
+      msg->AddText("Presence of a second peak around 40"); //Type of the Error, in SM XX
+      msg->AddText("If NOT a technical run,");
+      msg->AddText("call EMCAL on-call.");
+      h->SetFillColor(kRed);
+    } else if (checkResult == Quality::Medium) {
+      LOG(info) << "Quality::medium, setting to orange";
+      h->SetFillColor(kOrange);
+    }
+    h->SetLineColor(kBlack);
+  }
+  if (mo->getName() == "FECidMaxChWithInput_perSM" || "BunchMinRawAmplutudeFull_PHYS") {
```

Comment:
"BunchMinRawAmplutudeFull_PHYS" appears twice, I think this should be "PayloadSizeTFPerDDL"